dialect.go

```diff
@@ -72,6 +72,9 @@ type Dialect interface {
 	IfSchemaNotExists(command, schema string) string
 	IfTableExists(command, schema, table string) string
 	IfTableNotExists(command, schema, table string) string
+
+	// The command to create a new database/schema
+	CreateSchemaCommand() string
```

Comment:
Unfortunately, this would force a major version bump, because custom dialects would no longer implement the updated `Dialect` type.  Can we do this as an optional type that dialects are not required to implement (and then use the old logic as a fallback)?  Maybe something like `type NonstandardSchemaDialect interface { CreateSchemaCommand() string }`?

Reply:
Excellent idea. I'd like to change it slightly, though, since even a `NonstandardSchemaDialect` could vary amongst custom implementations. Maybe a `type SchemaCreator() interface { CreateSchemaCommand() string }` instead?

Reply:
I think `SchemaCreator` is a little misleading, since all other dialects will still be able to create schemas.  What we're providing is a way for dialects to inform `gorp` that they are unable to execute `create schema` operations per the SQL standard.

For those reasons, I still think something along the lines of "nonstandard" should be part of the name.  It is accurate - these are dialects that cannot handle the SQL standard for `create schema`, so they should be labeled as such.  `NonstandardSchemaCreator` sounds fine to me, too.

Also, we might want to pass the schema name in to `CreateSchemaCommand` so that dialects have more power over the resulting SQL command.

Reply:
@zikes Hey, just checking in; have you had any more time to work on this?